src/wwwroot/services/auth.js

```diff
@@ -253,7 +253,8 @@
           hourly: mapLimit(response.data.hourly),
           hasLimits: !response.data.noLimits,
           endDate: mapDate(response.data.endDate),
-          requiresDkimConfiguration: !!response.data.dkimConfigurationRequired
+          requiresDkimConfiguration: !!response.data.dkimConfigurationRequired,
+          requiresTrackingDomainConfiguration: !!response.data.trackingDomainConfigurationRequired
```

Comment:
Could you add a new subtask to Jira or update one in order to not forget to implement backend for this feature?

Reply:
http://jira.makingsense.com/browse/DR-1537